### PR TITLE
feat(earn): Show daily yield rate on the pool info screen if available

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

https://www.figma.com/design/E1rC3MG74qEg5V4tvbeUnU/Stablecoin-Enablement?node-id=7514-85993&node-type=instance&m=dev
https://linear.app/valora/issue/ACT-1406/daily-rate

* Daily yield rate in pool info screen when available
* small style fix to learn more link 

#### Daily Yield Rate

https://github.com/user-attachments/assets/9ad04f6c-a06c-4e49-9af5-a9d8c6a2d1f7

#### Link Style

https://www.figma.com/design/E1rC3MG74qEg5V4tvbeUnU/Stablecoin-Enablement?node-id=7514-85993&node-type=instance&m=dev
| Before | After |
| --- | --- |
|  |  |

### Test plan

Unit tests and manually verified

### Related issues

https://linear.app/valora/issue/ACT-1406/daily-rate

### CODE REVIEW

File: src/positions/types.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ interface EarnDataProps {
   depositTokenId: string
   withdrawTokenId: string
   rewardsPositionIds?: string[]
+  dailyYieldRatePercentage?: number // The daily yield rate percentage
```
Comment (finnian0826) [inline src/positions/types.ts:39]: nit: Is this comment necessary, just repeats what is in the name of the property

File: src/earn/EarnPoolInfoScreen.tsx [hunk 2]
```diff
@@ -573,6 +601,22 @@ export default function EarnPoolInfoScreen({ route, navigation }: Props) {
             tokensInfo={tokensInfo}
             earnPosition={pool}
           />
+          {!!dataProps.dailyYieldRatePercentage && dataProps.dailyYieldRatePercentage > 0 && (
```
Comment (satish-ravi) [inline src/earn/EarnPoolInfoScreen.tsx:604]: probably fine, but should we check its >= 0.00005? so we don't end up showing 0.0000 with the round with toFixed?

File: src/earn/EarnPoolInfoScreen.tsx
Comment (satish-ravi) [inline src/earn/EarnPoolInfoScreen.tsx:1]: This file is nearing 1000 lines.. I was thinking of breaking this up, I am already putting the safety card in a different file

**Review by satish-ravi — APPROVED** [review]

**Review by finnian0826 — COMMENTED** [review]
looks like root state schema needs to be updated but then lgtm!

**Pull request merged**